### PR TITLE
Add Suitecrm.com.xml

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

File: src/chrome/content/rules/Suitecrm.com.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!--
+	Nonfunctional hosts in *.suitecrm.com:
+
+	h: http redirect
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
```
Comment (pipboy96) [inline src/chrome/content/rules/Suitecrm.com.xml]: This entire comment may be removed.

**Comment by pipboy96** [comment]
```
forum.suitecrm.com
marketing.suitecrm.com
portal.suitecrm.com
webinar.suitecrm.com
```

File: src/chrome/content/rules/Suitecrm.com.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+<ruleset name="Suitecrm.com">
+	<target host="suitecrm.com" />
+	<target host="www.suitecrm.com" />
+	<target host="marketing.suitecrm.com" />
+	<target host="portal.suitecrm.com" />
+	<target host="sodsite.staging.suitecrm.com" />
+	<target host="wpsite.staging.suitecrm.com" />
+	<target host="store.suitecrm.com" />
+	<target host="wordpress.uat.suitecrm.com" />
```
Comment (ivysrono) [inline src/chrome/content/rules/Suitecrm.com.xml]: https://travis-ci.org/EFForg/https-everywhere/jobs/586533088#L324

Comment (J0WI) [reply src/chrome/content/rules/Suitecrm.com.xml]: Fixed

**Pull request merged**